tests/views/test_reporting_units.py

```diff
@@ -180,7 +177,7 @@ def test_get_reporting_unit_survey_fail(self, mock_request):
         response = self.client.get("/reporting-units/50012345678", follow_redirects=True)
 
         request_history = mock_request.request_history
-        self.assertEqual(len(request_history), 6)
+        self.assertEqual(len(request_history), 5)
```

Comment:
If it's making one less request, doesn't that mean we can remove one of the mocks defined in this test?
Can you check this for all the ones in this file that were reduced by 1.  The less mocks we have to set up, the better.